site/docs/4.6.0/overview/releaseNotes.md

```diff
@@ -2,16 +2,60 @@
 title: Apache BookKeeper 4.6.0 Release Notes
 ---
 
-[provide a summary of this release]
+This is the seventh release of BookKeeper as an Apache Top Level Project!
+
+The 4.6.0 release incorporates new fixes, improvements, and features since previous major release 4.5.0.
 
 Apache BookKeeper users are encouraged to upgrade to 4.6.0. The technical details of this release are summarized
 below.
 
 ## Highlights
 
-[List the highlights]
+The main features in 4.6.0 cover are around following areas:
+- Dependencies Upgrade
+- Bookie enhancement
+- BookKeeper Admin REST API
+- New BookKeeper API
+- Performance improvement
+- Deployment or Ease of use 
+
+### Dependencies Upgrade
+
+- Upgrade Protobuf to `3.4`.
+
```

Comment:
Needs a section for API breaks.
The one I've hit across is that BookKeeper constructor no longer throwing KeeperException. This breaks compilation on existing apps.
Also PlacementPolicies have had an interface change, which should be noted.
There may be more.

Reply:
Thanks, PR #822 made it.